src/storage/index/LookupProcessor.cpp

```diff
@@ -128,6 +134,12 @@ void LookupProcessor::onProcessFinished() {
                        [this](const auto& col) { return context_->tagName_ + "." + col; });
     }
     resp_.set_data(std::move(resultDataSet_));
+    if (FLAGS_profile_storage_detail) {
```

Comment:
Consider move this part to `BaseProcessor::onFinished`. `profile_detail_` could be a member of `BaseProcessor`.